nncf/common/tensor_statistics/statistic_point.py

```diff
@@ -68,20 +73,55 @@ def add_statistic_point(self, statistic_point: StatisticPoint):
             self.data[target_node_name].append(statistic_point)
 
     def iter_through_statistic_points_in_target_node(self, target_node_name: str,
-                                                     statistic_point_condition_func: Callable[
-                                                         [StatisticPoint], bool]):
+                                                     filter_fn: Callable[[StatisticPoint], bool]) -> StatisticPoint:
+        """
+        Returns iterable through all statistic points in node with target_node_name.
+
+        :param filter_fn: Callable to filter statistic containers according to its statistic point.
+        :return: Iterable through all statistic points in node with target_node_name.
+        """
         _statistic_points = self.data[target_node_name]
         for _statistic_point in _statistic_points:
-            if statistic_point_condition_func(_statistic_point):
+            if filter_fn(_statistic_point):
                 yield _statistic_point
 
+    def get_tensor_collectors(
+            self,
+            filter_fn: Optional[Callable[[StatisticPoint], bool]] = None) \
+            -> Generator[Tuple['Algorithm', StatisticPoint, TensorStatisticCollectorBase], None, None]:
```

Comment:
Just to test, please apply black formatting here. As far as I remember, we agreed on the black style as the default style for the project.

Reply:
Please check